core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -93,13 +107,80 @@
       "<code>MetricReporter</code> interface allows plugging in classes that will be notified " +
       "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+  public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
```

Comment:
Out of curiosity (and maybe @ijuma has a good idea here as well), how likely is it that we'd need to add any more configs here? I am looking at this list and wondering how much maintenance pain this could end up being between Kafka, this, the schema-registry kafkastore-prefixed versions, et al. In this repo, we don't depend on anything Kafka-related except for the tests you're adding here, but if there's a chance we need to add stuff, update docstrings, etc, we might want to figure out a strategy to manage all that info centrally and then provide it to the N different projects that we're now adding SSL support to.